.github/actions/setup-javascript/action.yml

```diff
@@ -6,7 +6,7 @@ runs:
     - name: Set up Bun
       uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2.0.1
       with:
-        bun-version: 1.0.11
+        bun-version: 1.1.34
```

Comment:
@johndeange Why specify any version of bun at all ?   Can't we just take lines 8 and 9 out altogether and let it use the version from the GHA natively has ?  Was this put in from when we had stuff breaking with cypress and bun playing nicely together and couldn't upgrade bun for a while ?

Reply:
I updated this to `latest` in the PR.  It looks like this was the initial version when we added bun.  As per our conversation, the default is probably `latest`

Reply:
yeah it is according to their docs and would be what's used if we didn't have those lines